src/app/containers/Include/amp/VjAmp.jsx

```diff
@@ -75,7 +75,8 @@ const StyledAmpIframe = styled(AmpIframe)`
       cursor: pointer;
       display: block;
       padding: ${GEL_SPACING} ${GEL_SPACING_DBL};
-      &:hover, &:focus {
+      &:hover,
+      &:focus {
```

Comment:
Is this intentional?

Reply:
yes, the bump to prettier resulted in some linting errors.

Reply:
Sorry! This looked like an addition, but it's just whitespace! All good, thanks 👍